common/Reducers.jsm

```diff
@@ -63,6 +63,7 @@ const INITIAL_STATE = {
     spocs: {
       spocs_endpoint: "",
       lastUpdated: null,
+      showSpocs: false,
```

Comment:
This is needed so the jsx gets a better more explicit handle on if spocs are on or not.